src/test/resources/features/acceptance/navigation.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Application process navigation is controlled
+  In order to help me complete the application correctly
+  As a potential claimant
+  I want to make sure that only the correct application flow can be used
+
+  Scenario Outline: Do you live in Scotland page is always the first page
+    Given I am starting a new application
+    When I navigate to the <page> page
+    Then I am shown the do you live in Scotland page
+    Examples:
+      | page                            |
+      | enter name                      |
+      | enter national insurance number |
+      | enter date of birth             |
+
+# TODO Enable more scenarios as the navigate functionality is added
```

Comment:
I thought this was ok to leave commented out and uncomment them as they get implemented, but I'm happy to remove if anyone objects.